<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow
{
/*
background-image: url(&quot;D:/Coding Stuff/qt apps/BabyPS/Assets/bg.jpg&quot;);
background-repeat: no-repeat;
background-position: center;
*/
	background-color: rgb(245, 246, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton{background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 white, stop: 1 grey);
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 10px;}*/
QPushButton {
    background-color: rgb(232, 234, 255);
	border: 0px solid black;
	font: 700 12pt &quot;Segoe UI&quot;;
	color: rgb(91,102,247);
	border-radius: 10px;
}
QPushButton:hover {
    background-color: rgb(220, 219, 235);
}

QPushButton:pressed {
    background-color: rgb(91,102,247);
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);  
}</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(232, 234, 255);
	border: 0px solid black;
	font: 700 12pt &quot;Segoe UI&quot;;
	color: rgb(91,102,247);
	border-radius: 10px;
}
QPushButton:hover {
    background-color: rgb(220, 219, 235);
}

QPushButton:pressed {
    background-color: rgb(91,102,247);
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);  
}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_encrypt">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_decrypt">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>55</y>
      <width>661</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Empty File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>210</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Merge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="countWord">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Count Word</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Upper Case</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Lower Case</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_8">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>460</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>1st Caps</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>505</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Words :</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordsNum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>505</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>505</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Lines:</string>
    </property>
   </widget>
   <widget class="QLabel" name="linesNum">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>505</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textSeacrh">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="seacrhLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>544</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter The Word Here:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>540</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 10px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="count_btn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>540</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(65, 77, 246);
	border: 0px solid black;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255,255,255);
	border-radius: 10px;
}
QPushButton:hover {
    background-color: rgb(30, 45, 211);
}

QPushButton:pressed {
    background-color: rgb(65, 77, 246);     
}</string>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="countLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>544</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter The Word Here:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textCount">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
